src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -567,6 +567,11 @@ protected View getWelcomeView() {
         return new MultiBranchProjectEmptyView(this);
     }
 
+    @Override
+    public View getView(String name) {
+        return getPrimaryView();
```

Comment:
The behavior described in that Jira issue assumes the user indexed some branches or pull requests. In this case, you don't have a empty view.